flink-core/src/test/java/org/apache/flink/api/common/io/GlobFilePathFilterTest.java

```diff
@@ -138,4 +140,27 @@ public void doubleStarPattern() {
 		assertFalse(matcher.filterPath(new Path("a/b")));
 		assertFalse(matcher.filterPath(new Path("a/b/c")));
 	}
+
+	@Test(expected = NullPointerException.class)
+	public void testIncluePatternIsNull() {
+		new GlobFilePathFilter(
+			null,
+			Collections.<String>emptyList());
+	}
+
+	@Test(expected = NullPointerException.class)
+	public void testExcludePatternIsNull() {
+		new GlobFilePathFilter(
+			Collections.singletonList("**"),
+			null);
+	}
+
+	@Test
+	public void testGlobFilterSerializable() throws IOException {
+		GlobFilePathFilter matcher = new GlobFilePathFilter(
+			Collections.singletonList("**"),
+			Collections.<String>emptyList());
+
+		CommonTestUtils.createCopySerializable(matcher);
```

Comment:
A doof addition here would be to check that after the serialization, the filter behaves in the same way as before.